test/test_database.py

```diff
@@ -201,7 +201,7 @@ def test_list_collection_names_filter(self):
         db.capped.insert_one({})
         db.non_capped.insert_one({})
         self.addCleanup(client.drop_database, db.name)
-
+        filter: dict
```

Comment:
revert?

Reply:
Oh I see. I thought it was a copy/past error. Down below there's already `filter: Any`. Can we define this once? perhaps `filter: Union[None, dict]` would be better?